src/scene/shader-lib/chunks-wgsl/common/frag/envAtlas.js

```diff
@@ -1,4 +1,8 @@
 export default /* wgsl */`
+
+#ifndef _ENVATLAS_INCLUDED_
```

Comment:
any reason for _'s? don't think other defines around the codebase have these.

Reply:
I guess just a habit from c++ days, to separate those double-include guards from normal defines (that people can depend on) to avoid possible collision. We already have _DECODE_INCLUDED_ in code as well.